plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -198,6 +198,20 @@ chain_plugin::~chain_plugin(){}
 
 void chain_plugin::set_program_options(options_description& cli, options_description& cfg)
 {
+   // build wasm_runtime help text
+   std::string wasm_runtime_opt = "Override default WASM runtime(\"eos-vm\"";
+   std::string wasm_runtime_desc = "\"eos-vm\" : (Default runtime) A WebAssembly interpreter providing deterministic and efficient time bound execution with debugging support.\n";
```

Comment:
I believe the default is eos-vm-jit now

Reply:
Changed the default to eos-vm-jit.
  --wasm-runtime runtime                Override default WASM runtime ( 
                                        "eos-vm-jit", "eos-vm")
                                        "eos-vm-jit" : (Default runtime) A 
                                        WebAssembly compiler that takes WASM 
                                        and generates native code using a just 
                                        in time compiler.
                                        "eos-vm" : A WebAssembly interpreter 
                                        providing deterministic and efficient 
                                        time bound execution with debugging 
                                        support.